ons_alpha/release_calendar/models.py

```diff
@@ -80,7 +81,14 @@ class ReleasePage(BasePage):
         related_name="+",
     )
 
-    is_accredited = models.BooleanField("Accredited Official Statistics", default=False)
+    is_accredited = models.BooleanField(
+        "Accredited Official Statistics",
+        default=False,
+        help_text=(
+            "If ticked, will display an information block about the data being "
+            "accredited official statistics and include the accredited logo"
+        ),
```

Comment:
Nit: This help text is a little hard to read - perhaps it can be reworded to be simpler? Instead of saying what the setting will do, perhaps just:

```suggestion
        help_text=(
            "Has this been accredited 'Official Statistics'?"
        ),
```